src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+
+import { ErrorType } from '../../types/ErrorType';
+import { errorMessages } from '../../common/constants';
+
+type Props = {
+  error: ErrorType;
+  onClose: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = React.memo(({
+  error,
+  onClose,
+}) => {
+  const errorMessage = errorMessages[error];
+
+  useEffect(() => {
+    const timerId = setInterval(onClose, 3000);
+
+    return () => clearInterval(timerId);
+  }, [onClose]);
```

Comment:
Why do you need interval for it? Use timeout instead
Also better to reset error in App component (create one method that will add and reset error message) and render this component only when there is some errorMessage